### DISCUSSION TITLE
Quasar support for Nuxt 3

### DISCUSSION THREAD

**Discussion by JorgenSQ** [quasarframework/quasar#11684] [Ideas / Proposals]
Are there any plans to ever support nuxt 3 compatibility for Quasar?

**Comment by smolinari** [comment]
@JorgenSQ - Quasar practically does what Nuxt does, so there is no reason to work it into Nuxt. 

The only thing Quasar currently doesn't have is static site generation (SSG) or automated routing. If you want that, you can use Quasar as a Vue plugin and make use of Quasar's components in Nuxt. But, then you'd be stuck with just SPA, SSR and SSG and can't do anything else that Quasar can do. 

Scott

**Reply by hawkeye64** [reply]
There is also an SSG app extension available.

**Reply by JorgenSQ** [reply]
In that case, thats great news!

What I am looking for in nuxt 3 specifically is the ability to write server-side code / middleware much like next.js for react. 
(https://v3.nuxtjs.org/docs/directory-structure/server)

I might be blind, but I have not found that option in Quasar?

**Reply by hawkeye64** [reply]
https://quasar.dev/quasar-cli/developing-ssr/ssr-middleware#introduction
